CDCgov/prime-reportstream
# 5753 Generate HL7 TQ1 segment from FHIR

This PR generates the TQ1 segment from FHIR data. 

Test Steps:
1. Check out the branch for this PR.
2. Run the following command to convert a FHIR bundle to HL7.  
`./prime fhirdata --input-file "src/testIntegration/resources/datatests/HL7_to_FHIR/sample_co_1_20220518-0001.fhir" -s metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml`
3. Verify an HL7 message is generated from the conversion and includes the TQ1 segments.
4. Look at the source HL7 used at `src/testIntegration/resources/datatests/HL7_to_FHIR/sample_co_1_20220518-0001.hl7` and verify the data for the TQ1 segments are identical.

## Changes
- Added config for the FHIR to HL7 conversion of the TQ1 segment

## Checklist

### Testing

### Process

## Linked Issues
- #5753

## To Be Done

## Specific Security-related subjects a reviewer should pay specific attention to
